src/components/NewbieSelect/__tests__/__snapshots__/NewbieSelect.tsx.snap

```diff
@@ -1,7 +1,32 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Component - NewbieSelect renders correctly 1`] = `
-<h1>
-  Newbie Select
-</h1>
+Array [
+  <h1>
+    Newbie Select
+  </h1>,
+  <button
+    className="MuiButtonBase-root MuiFab-root MuiFab-sizeMedium MuiFab-primary"
+    disabled={false}
+    onBlur={[Function]}
+    onDragLeave={[Function]}
+    onFocus={[Function]}
+    onKeyDown={[Function]}
+    onKeyUp={[Function]}
+    onMouseDown={[Function]}
+    onMouseLeave={[Function]}
+    onMouseUp={[Function]}
+    onTouchEnd={[Function]}
+    onTouchMove={[Function]}
+    onTouchStart={[Function]}
+    tabIndex={0}
+    type="button"
+  >
+    <span
+      className="MuiFab-label"
+    >
+      +
+    </span>
+  </button>,
```

Comment:
As the "+" button has its own snapshot, would it makes sense to mock it here?

Reply:
It's fixed here 6968d4e7e8338a839706f0f60446463ecc90d6db and df0c5f9ab825a6913ccc7a4e0e1e2f2d2dc7f045 in my PR.